client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -1,24 +1,40 @@
 import { useTranslate } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import earnCardPrompt from 'calypso/assets/images/customer-home/illustration--secondary-earn.svg';
+import { COURSE_SLUGS } from 'calypso/data/courses';
+import { useRouteModal } from 'calypso/lib/route-modal';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import EducationalContent from '../educational-content';
+import PaymentsFeaturesModal from './payments-features-modal';
 
 export const EDUCATION_EARN = 'home-education-earn';
 
 const EducationEarn = ( { siteSlug } ) => {
 	const translate = useTranslate();
+	const { isModalOpen, openModal, closeModal } = useRouteModal(
+		'coursePaymentsModal',
+		COURSE_SLUGS.PAYMENTS_FEATURES
+	);
 
 	return (
 		<EducationalContent
 			title={ translate( 'Make money from your website' ) }
 			description={ translate(
 				'Accept credit and debit card payments on your website for just about anything.'
 			) }
-			links={ [
+			modalLinks={ [
 				{
 					calypsoLink: true,
 					url: `/earn/${ siteSlug }`,
+					ModalComponent: PaymentsFeaturesModal,
+					modalComponentProps: {
+						isVisible: isModalOpen,
+						onClose: ( event ) => {
+							event.preventDefault();
+							closeModal();
+						},
+					},
+					onClick: () => openModal(),
```

Comment:
I can't remember: did we get an agreement with Marketing about replacing the CTA of the existing "Make Money" card in My Home from navigating to `/earn` to opening the new Payments course?

I don't have a strong opinion either way, but I wonder if those cards should be separate. 🤔 

To give a sense, these are some handpicked cards:

| Card | CTA Target |
| - | - |
| <img width="704" alt="Screenshot 2022-08-30 at 11 15 45" src="https://user-images.githubusercontent.com/2070010/187412350-33f7d8f0-a7b8-4e88-add2-5f3025262050.png"> | Open Blogging course |
| <img width="704" alt="Screenshot 2022-08-30 at 11 15 50" src="https://user-images.githubusercontent.com/2070010/187412445-569c0c9e-f90d-4c99-939c-bf2dc4368538.png"> | Open support modal about Pexels |
| <img width="704" alt="Screenshot 2022-08-30 at 11 15 32" src="https://user-images.githubusercontent.com/2070010/187412199-2f15a41a-528e-4d5d-8879-65423b12a127.png"> | Before this PR: open `/earn`<br>After: open Payments course |

I guess we have some options here:
1. Reuse the existing card, rewriting it. At the very least, we should replace "Start making money" with "Learn more" (although then the rest of the card will make little sense 😄 ).
2. Create a separate card with its own copy.
3. Add a secondary "Learn more" CTA to the existing card. (I haven't seen examples of cards with multiple links, but the `modalLinks` prop accepts an array, so it's at least allowed technically).

cc @sethmclellana8c @jeremyanderberg

Reply:
I like option 1 or 3, depending of how much traffic does the current link generate (option 1 if low traffic, and option 3 otherwise).

@lukaszwilczakwilnet can you check how many users click on the current "Start making money" link? That might help us to make a better informed decision. 

Seems [we're tracking those clicks](https://github.com/Automattic/wp-calypso/blob/a2cc6fa5ee914e53e75e8eaf147bf1bac549b5e4/client/my-sites/customer-home/cards/education/educational-content/index.jsx#L12-L17) so should be able to get that data from a Tracks Funnel (see PCYsg-5hd-p2).

Reply:
Event calypso_customer_home_education track 70-100 clicks per day but it's shared between all cards. Card "earn" is 19%. So simple calculation: it's 15-20 clicks per day, so 450-600 clicks per month.

I like option 3 most. Option 1 is also fine, but i think option 2 is some kind of duplication. 

@mmtr I don't have perspective yet if those numbers above are big or small, which option do you recommend?

Reply:
I like option 3 the most too, plus it has the advantage of (possibly) not requiring much additional work, which is always nice. 😄

Reply:
Yeah, I think option 3 is the most preferable one too.